changelog.d/9609.feature

```diff
@@ -0,0 +1 @@
+Added SsoAttributeRequirements to OpenID Connect-based SSO providers. Contributed by Hubbe King.
```

Comment:
Better if we phrase this in a way that someone not familiar with the code can understand:

```suggestion
Logins using OpenID Connect can require attributes on the `userinfo` response in order to login. Contributed by Hubbe King.
```